src/semver/version.py

```diff
@@ -78,31 +48,9 @@ def wrapper(self: "Version", other: Comparable) -> bool:
     return wrapper
 
 
-def _nat_cmp(a, b):  # TODO: type hints
-    def convert(text):
-        return int(text) if re.match("^[0-9]+$", text) else text
-
-    def split_key(key):
-        return [convert(c) for c in key.split(".")]
-
-    def cmp_prerelease_tag(a, b):
-        if isinstance(a, int) and isinstance(b, int):
-            return cmp(a, b)
-        elif isinstance(a, int):
-            return -1
-        elif isinstance(b, int):
-            return 1
-        else:
-            return cmp(a, b)
-
-    a, b = a or "", b or ""
-    a_parts, b_parts = split_key(a), split_key(b)
-    for sub_a, sub_b in zip(a_parts, b_parts):
-        cmp_result = cmp_prerelease_tag(sub_a, sub_b)
-        if cmp_result != 0:
-            return cmp_result
-    else:
-        return cmp(len(a), len(b))
+def _cmd(a, b):  # TODO: type hints
```

Comment:
I believe you meant `_cmp` 😉

Reply:
Oh yes, of course! Will correct it. Thanks for spotting this stupid error. :+1: